### PR TITLE
ROSE-231: Call /account/coins endpoint to add account coin info in coin storage

### PR DESCRIPTION
**Pull request by shiatcb**
Fixes # .
https://jira.coinbase-corp.com/browse/ROSE-231 

### Motivation
'Insufficient fund' error for prefunded_account in UTXO chain

### Solution
See PR

### CODE REVIEW

File: utils/utils.go [hunk 1]
```diff
@@ -491,6 +499,82 @@ func GetAccountBalances(
...
+		}
+
+		acctCoins = append(acctCoins, resp)
+	}
+
+	return acctCoins, nil
+}
+
+// getAccountCoins returns Rosetta type of response from /account/coins endpoint.
+func getAccountCoins(
```
Comment (xiaying-peng) [inline utils/utils.go:551]: Why we need this extra function?

File: utils/utils.go [hunk 2]
```diff
@@ -491,6 +499,82 @@ func GetAccountBalances(
...
+	IncludeMempool bool
+}
+
+// AccountCoins defines an account's coins info in the tip.
+type AccountCoinsResponse struct {
+	Coins []*types.Coin
+}
+
+// GetAccountCoins calls /account/coins endpoint and returns an array of coins in the tip.
+func GetAccountCoins(
```
Comment (xiaying-peng) [inline utils/utils.go:520]: Who will call this function?

Comment (xiaying-peng) [reply utils/utils.go:520]: Oh I see, your later PR in rosetta-cli uses this.

File: utils/utils.go (hunk 1 above)
Comment (shiatcb) [reply utils/utils.go:551]: I think you are right, I'll remove this function.

File: utils/utils.go (hunk 2 above)
Comment (shiatcb) [reply utils/utils.go:520]: Yes https://github.com/coinbase/rosetta-cli/pull/290/files

File: utils/utils.go (hunk 1 above)
Comment (shiatcb) [reply utils/utils.go]: Done

**Review by xiaying-peng — APPROVED** [review]

**Pull request merged**